### PR TITLE
chore: [IA-177] Alert user when is paying and there are no payable methods

### PR DESCRIPTION
**Pull request by Undermaken**
## Short description
This PR shows an alert when the user is going to pay but he/she has no payable methods.
The alert is shown on transaction summary screen. It can be reached from 

- message
- payment data manual insertion
- QRcode reading

### iOS - from message
https://user-images.githubusercontent.com/822471/127862772-326c5145-34fa-4817-9558-fab687237e18.mp4

### iOS - manual insertion
https://user-images.githubusercontent.com/822471/127888691-214eec4f-adfc-4d64-bf6f-bb2d1ff8472f.mp4

### iOS - QRcode
https://user-images.githubusercontent.com/822471/127862862-4b891ef8-90f0-47d6-ac24-f91b0bb3b82d.mp4

### how to test
- using io-dev-server: empty your payment methods (set the config as shown in the screenshoort - http://127.0.0.1:3000) or make them all not payable (put `false` [on this line](https://github.com/pagopa/io-dev-api-server/blob/b1465706cd5cb69e93a3e032cbc3c5b7ece96622/src/routers/walletsV2/index.ts#L141))

Schermata 2021-08-02 alle 14 33 31

- [x] implement changes
- [x] add tests
- [x] track this scenario
- [x] add track action in the document

### CODE REVIEW

File: locales/it/index.yml [hunk 1]
```diff
@@ -808,6 +808,12 @@ datetimes:
   todayAt: oggi, alle
   notValid: Data non valida.
 payment:
+  alertNoPaymentMethods:
+    title: Aggiungi un metodo per effettuare pagamenti in app
+    message: Il metodo verrà salvato nel Portafoglio, così la prossima volta potrai pagare più velocemente.
+    buttons:
+      ok: Continua
+      ko: Annulla
```
Comment (pietro909) [inline locales/it/index.yml:816]: A basic translation for the German dictionary could be:

```
alertNoPaymentMethods:
    title: Bitte hinzufügen Sie einen Zahlungsdienstleister, um aus der IO-App zu bezahlen
    message: Der Zahlungsmethode wird in Ihrem Konto Gespeichert, damit Sie nächster Mal schneller bezahlen können.
    buttons:
      ok: Weiter
      ko: Abbrechen
```

My German is pretty rusty, though it should be better than nothing :-)

Comment (Undermaken) [reply locales/it/index.yml:816]: @pietro909 if it's ok for @thisisjp could you add a commit with the proposal locale?

File: locales/en/index.yml [hunk 2]
```diff
@@ -790,6 +790,12 @@ datetimes:
   todayAt: today, at
   notValid: Invalid date.
 payment:
+  alertNoPaymentMethods:
+    title: Add a method to make payments in the app
```
Comment (thisisjp) [inline locales/en/index.yml:794]:
```suggestion
    title: Add a method to make in-app payments
```

File: locales/en/index.yml [hunk 3]
```diff
@@ -790,6 +790,12 @@ datetimes:
...
+    message: The method will be saved in the Wallet, so you can pay faster next time.
```
Comment (thisisjp) [inline locales/en/index.yml:795]:
```suggestion
    message: The method will be saved in your Wallet, so you can pay easier next time.
```

File: locales/it/index.yml [hunk 4]
```diff
@@ -808,6 +808,12 @@ datetimes:
   todayAt: oggi, alle
   notValid: Data non valida.
 payment:
+  alertNoPaymentMethods:
+    title: Aggiungi un metodo per effettuare pagamenti in app
+    message: Il metodo verrà salvato nel Portafoglio, così la prossima volta potrai pagare più velocemente.
```
Comment (thisisjp) [inline locales/it/index.yml:813]:
```suggestion
    message: Il metodo verrà salvato nel Portafoglio, così la prossima volta potrai pagare più facilmente.
```

**Review by thisisjp — COMMENTED** [review]
@Undermaken the flow looks slightly different from what designed—it makes sense to let users freely explore the "pay" feature before adding at least a payment method, but I still find it quite frustrating when adding numbers manually, though... can we at least anticipate the alert in case of manual typing? (Meanwhile, you already got a silver ux medal 🥈 )

File: locales/it/index.yml (hunk 1 above)
Comment (pietro909) [reply locales/it/index.yml:816]: https://github.com/pagopa/io-app/pull/3272/commits/0cef0c0b8fa623c23c4b5e5c109c83939c2ac44a

**Comment by Undermaken** [comment]
> @Undermaken the flow looks slightly different from what designed—it makes sense to let users freely explore the "pay" feature before adding at least a payment method, but I still find it quite frustrating when adding numbers manually, though... can we at least anticipate the alert in case of manual typing? (Meanwhile, you already got a silver ux medal 🥈 )

@thisisjp I point the gold
check the updated video in the PR description

**Review by thisisjp — APPROVED** [review]

**Review by pietro909 — APPROVED** [review]

**Pull request merged**